src/DistributedFESpaces.jl

```diff
@@ -2,6 +2,7 @@ struct DistributedFESpace{V} <: FESpace
   vector_type::V
   spaces::DistributedData{<:FESpace}
   gids::DistributedIndexSet
+  model::DistributedDiscreteModel
```

Comment:
Is really necessary to include this field here? In some situations is possible to construct a FE Space without a model (e.g., a DG space from a triangulation). Thus, we need to remove this field.

Reply:
No, (I realized after your comment) it is not. I was assuming as a dogma the following statement: *"it has to be possible to create any type extension of  `AssemblyStrategy` solely from the local portion of a `DistributedFESpace`"*. But this is clearly not the case as the signature of an outer constructor of a type extension of  `DistributedAssemblyStrategy` is not fixed, so that we have 100% freedom here. (Note that `OwnedCellsStrategy` requires access to the corresponding `IndexSet` portion of the `DistributedIndexSet` object associated to the distributed model).